src/core/get.spec.ts

```diff
@@ -13,6 +13,12 @@ describe('(v) get', () => {
     strictEqual(x.data[2], x.get(2));
   });
 
+  it('should work with ndim=0', () => {
+    const x = array([13]).reshape();
+    strictEqual(13, x.get());
```

Comment:
Is there a reason this should work? I would expect `x.get()` to return an error here instead of returning 13. The proper way to get the value for this array would be `x.get(0)`.

Reply:
I think you missed the `reshape()`. `x` has shape `[]`, so it doesn't *have* a first axis to index.